lib/browser/api/menu-item.js

```diff
@@ -7,18 +7,8 @@ let nextCommandId = 0
 const MenuItem = function (options) {
   const {Menu} = require('electron')
 
-  this.selector = options.selector
-  this.type = options.type
-  this.role = options.role
-  this.label = options.label
-  this.sublabel = options.sublabel
-  this.accelerator = options.accelerator
-  this.icon = options.icon
-  this.enabled = options.enabled
-  this.visible = options.visible
-  this.checked = options.checked
-
-  this.submenu = options.submenu
+  Object.assign(this, options)
```

Comment:
This is scary for a few reasons.  Mostly because what happens if the user does something silly like pass in `overrideProperty` as a property on the `options` object.

This would causes the `overrideProperty` method to be overriden (oh how ironic).  Can we blacklist that method from being overriden 👍